src/app/(home)/aboutUs/page.tsx

```diff
@@ -1,5 +1,33 @@
+import Footer from "../../../components/mainLayout/footer/footer";
+
+import AboutUsCard from "../../../components/mainLayout/aboutUsCard/aboutUsCard";
+
+import teamList from "../../../mocks/AboutUsTeam.json";
+
+import { IAboutUsCard } from "../../../types/common/ComponentsProps";
+
+import style from "./aboutUs.module.scss";
+
 function AboutUs() {
-	return <div />;
+	return (
+		<div className={style.aboutUsWrap}>
+			<div className={style.aboutUsContainer}>
+				<div className={style.aboutUsMain}>
+					<div className={style.aboutUsTitle}>Классные ребята с горящими глазами.</div>
+					<div className={style.aboutUsCards}>
+						{teamList && teamList?.team.length ? (
+							teamList.team.map(({ teamRole, photo }: IAboutUsCard) => (
+								<AboutUsCard photo={photo} teamRole={teamRole} key={teamRole} />
+							))
+						) : (
+							<div className={style.aboutUsTitle}>Команда скоро вернётся</div>
+						)}
```

Comment:
better move it to separate fucntion and use here

Reply:
Ok. I will do